apps/metadata-editor/src/app/dashboard/sidebar/sidebar.component.spec.ts

```diff
@@ -34,4 +46,25 @@ describe('SidebarComponent', () => {
   it('should create', () => {
     expect(component).toBeTruthy()
   })
+
+  describe('logOut', () => {
+    it('should log out', async () => {
+      jest.spyOn(window, 'fetch').mockResolvedValue({
+        ok: true,
+      } as Response)
+
+      delete window.location
+      Object.defineProperty(window, 'location', {
+        value: { href: 'http://app-url.com' },
+        writable: true,
+      })
+
+      await component.logOut()
+
+      expect(window.fetch).toHaveBeenCalledWith(service.logoutUrl, {
+        method: 'GET',
+      })
+      expect(window.location.href).toBe('http://localhost:4200')
```

Comment:
Same here, we should use the actual host

Reply:
Well I think in the e2e it's always going to be localhost right?